src/translations/sk.json

```
@@ -0,0 +1,41 @@
+{
+    "log_in": "Prihlásiť sa",
+    "log_out": "Odhlásiť sa",
+    "loged_in_as": "Prihlásený ako",
+    "logged_in": "Prihlásený",
+    "logging_in": "Prihlásenie",
+    "logging_out": "Odhlásenie",
+    "sign_up": "Zaregistrovať sa",
+    "signing_up": "Registrácia",
+    "remember_password": "Zabudli ste heslo?",
+    "recover_password": "Obnoviť heslo",
+    "send_recovery_email": "Odoslať e-mail pre obnovenie",
+    "sending_recovery_email": "Odosielanie e-mailu pre obnovenie",
+    "never_mind": "Nevadí",
+    "update_password": "Aktualizovať heslo",
+    "updated_password": "Aktualizácia heslá",
+    "complete_your_signup": "Dokončite registráciu",
+    "site_url_title": "Nastavenie vývoja",
+    "site_url_link_text": "Vymazať URL localhost",
+    "site_url_message": "Zdá sa, že používate lokálny server. Povedzte nám adresu URL svojho Netlify servera.",
+    "site_url_label": "Zadajte URL svojho servera Netlify",
+    "site_url_placeholder": "URL vášho Netlify servera",
+    "site_url_submit": "Nastaviť adresu URL",
+    "message_confirm": "Na váš e-mail bol odoslaný odkaz na potvrdenie, pokračujte kliknutím na tento odkaz.",
+    "message_password_mail": "Poslali sme vám e-mail pre obnovenie hesla, heslo obnovíte kliknutím na odkaz v e-mailu.",
+    "message_email_changed": "Vaša e-mailová adresa bola aktualizovaná!",
+    "message_verfication_error": "Pri overovaní vášho účtu došlo k chybe. Skúste to znova alebo kontaktujte správcu.",
```

I like the more polite tone in the original English translation of the copy, so here's a suggestion of retaining it:

```suggestion
    "message_verfication_error": "Pri overovaní vášho účtu došlo k chybe. Prosím, skúste to znova alebo kontaktujte správcu.",
```